pika/adapters/libev_connection.py

```diff
@@ -99,6 +99,7 @@ def __init__(self,
             with warnings.catch_warnings():
                 warnings.simplefilter("ignore", RuntimeWarning)
                 self.ioloop = pyev.default_loop()
+                self.ioloop.update()
```

Comment:
@gst, this line will likely have a minor merge conflict with the same line in your PR #678. I needed this here in order to get a clean build.